en/simulation/ignition_gazebo.md

```diff
@@ -1,11 +1,11 @@
-# Ignition Gazebo Simulation
+# Gz Simulation
```

Comment:
We agree on calling it "Gazebo" and `gz` is the build target name for this new version

Reply:
Agreed